src/hooks/use-lesson.ts

```diff
@@ -5,7 +5,7 @@ import { SUBROUTES } from 'router/routes';
 import { LOADING_PROCESS_MAP, ProcessEnum } from 'utils/constants';
 import { SerializedError } from 'api/core';
 import { getNextOrPrevRoute } from 'utils/get-next-or-prev-route';
-import { LessonModel, UserLessonProgress } from 'api/lessons';
+import { LessonModel, WebinarModel, UserLessonProgress } from 'api/lessons';
```

Comment:
```suggestion
import { LessonModel, LessonType, WebinarModel, UserLessonProgress } from 'api/lessons';
```

а 22 строку `import { LessonType } from '../api/lessons/types';` удалить